### ISSUE TITLE
Blury Hover Font

### ISSUE THREAD

**Issue by Foggalong** [ozonos/ozon-shell-theme#9]
Whatever effect is being used to change the hover colour mens that the text becomes jagged and unclear.

![screenshot from 2014-12-06 17 33 00](https://cloud.githubusercontent.com/assets/5920259/5328299/3d9f499e-7d6e-11e4-999c-408ad436f4b9.png)

**Comment by me4oslav** [comment]
I can not confirm it? Does this still happen to you?

**Comment by Foggalong** [comment]
Yeah this is still the case. My guess would be that it only effect no HD screen then :/
